node/src/accountant/db_access_objects/utils.rs

```diff
@@ -334,14 +342,23 @@ pub fn remap_receivable_accounts(accounts: Vec<ReceivableAccount>) -> Vec<UiRece
         .map(|account| UiReceivableAccount {
             wallet: account.wallet.to_string(),
             age_s: to_age(account.last_received_timestamp),
-            balance_gwei:{
-                let gwei =  (account.balance_wei / (WEIS_IN_GWEI as i128)) as i64;
-                if gwei != 0 {gwei} else {panic!("Broken code: ReceivableAccount with balance \
-                 between {} and 0 gwei passed through db query constraints; wallet: {}, balance: {}",
-                                                 if account.balance_wei.is_positive() {"1"}else{"-1"},
-                                                 account.wallet,
-                                                 account.balance_wei
-                )}
+            balance_gwei: {
+                let gwei = (account.balance_wei / (WEIS_IN_GWEI as i128)) as i64;
+                if gwei != 0 {
+                    gwei
+                } else {
+                    panic!(
+                        "Broken code: ReceivableAccount with balance between {} and 0 gwei \
```

Comment:
You may prefer to use the same code for conversion. If you decide to do so then prefer this piece of code over the other.